symex-primitives.el

```diff
@@ -169,8 +169,12 @@
   "Check if the symex is an unquoted list."
   (looking-at (concat "," lispy-left)))
 
+(defun symex--clojure-deref-reader-macro-p ()
+  "Check if the symex is a clojure deref reader macro."
+  (looking-at (concat "@" lispy-left)))
+
 (defun symex--clojure-literal-lambda-p ()
-  "Check if the symex is a clojurescript anonymous function literal."
+  "Check if the symex is a clojure anonymous function literal."
```

Comment:
Thinking about this some more; I believe that 'Clojure' is understood to be the overarching name of the language and can be read to include dialects. Sticking with this naming approach will keep docstrings a little bit shorter.

Reply:
super nit: probably capitalize "Clojure"